tests/functional/rhcs_setup.yml

```diff
@@ -27,22 +27,10 @@
         dest: "{{ change_dir }}/vagrant_variables.yml"
       when: change_dir is defined
 
-    - name: change ceph_stable to False
+    - name: change ceph_origin to distro
       replace:
-        regexp: "ceph_stable:.*"
-        replace: "ceph_stable: False"
-        dest: "{{ change_dir }}/group_vars/all"
-      when: change_dir is defined
-
-    - name: set ceph_rhcs to True
-      lineinfile:
-        line: "ceph_rhcs: True"
-        dest: "{{ change_dir }}/group_vars/all"
-      when: change_dir is defined
-
-    - name: set ceph_origin to distro
-      lineinfile:
-        line: "ceph_origin: distro"
+        regexp: "ceph_origin:.*"
+        replace: "ceph_origin: distro"
```

Comment:
You also need to set ``ceph_repository: rhcs`` in ``rhcs_setup.yml``.  Can ``ceph_origin: distro`` and ``ceph_repository: rhcs`` work together? This is certainly needed.